presto-hive/src/main/java/io/prestosql/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -373,6 +378,11 @@ private void invokeNoMoreSplitsIfNecessary()
             if (tableBucketInfo.isPresent()) {
                 throw new PrestoException(NOT_SUPPORTED, "Presto cannot read bucketed partition in an input format with UseFileSplitsFromInputFormat annotation: " + inputFormat.getClass().getSimpleName());
             }
+
+            if (AcidUtils.isTransactionalTable(table.getParameters())) {
+                throw new PrestoException(NOT_SUPPORTED, "Presto cannot read hive transactional tables in an input format with UseFileSplitsFromInputFormat annotation: " + inputFormat.getClass().getSimpleName());
```

Comment:
This cannot be support in Presto right now due to the way we generate the splits for transactional tables. I think this would also be a invalid configuration but I will reword the message as suggested.